docs/middlewares.md

```diff
@@ -487,6 +487,7 @@ For each secret, you also provide the name under which its value should be added
 - `secrets` (object) : Map of secrets to fetch from Secrets Manager, where the key is the destination, and value is secret name in Secrets Manager.
   Example: `{secrets: {RDS_LOGIN: 'dev/rds_login'}}`
 - `awsSdkOptions` (object) (optional): Options to pass to AWS.SecretsManager class constructor.
+- `shouldFail` (boolean) (optional): Defaults to `false`. Set it to `true` if you want your lambda to fail in case call to AWS Secrets Manager fails (secrets don't exist or internal error)
```

Comment:
Maybe name it more explicitly to better convey the intent?

Like `throwOnMissingParam` or `ignoreMissingParams`

Reply:
Yeah I couldn't think of any good name, your proposal doesn't cover errors other than missing param in store - how about `throwOnFailedCall`?

Reply:
Hmm, agree, doesn't cover throttling errors etc, so another thing I can think of is to borrow [bail](https://jestjs.io/docs/en/cli#bail) name from jest

Reply:
Hmm, I think `throw` would be more descriptive here, `bail` is not so common